### ISSUE TITLE
npm ERR! Maximum call stack size exceeded

### ISSUE THREAD

**Issue by Zougi** [Medium/phantomjs#605]
npm shrinkwrap fails with maximum call stack size exceeded on phantomjs-prebuilt 2.1.12

```sh
13:30:28 INFO: npm ERR! node v4.4.7
13:30:28 INFO: npm ERR! npm  v3.10.6
13:30:28 INFO: 
13:30:28 INFO: npm ERR! Maximum call stack size exceeded
13:30:28 INFO: npm ERR! 
```

**Comment by Zougi** [comment]
nodejs LTS v4.5.0 just came out. Changelog tells circular references are now allowed. maybe this fix this issue. I can't test right away

**Comment by franher** [comment]
@Zougi take a look on https://github.com/npm/npm/issues/13327 is resolved on npm@3.10.7

**Comment by Zougi** [comment]
Yup, I linked the issue too. I do not own the VM I'm building against. Therfore I cannot update to latest npm version, this is why I opened the issue. I finally used npm shrinkwrap to use phantomjs-prebuilt 2.1.11. I let this issue open in case a phantomjs maintainer do not already know phantomjs uses circular dependencies and want to get rid of them.
